### ISSUE TITLE
数据预处理问题。。。

### ISSUE THREAD

**Issue by duan348733684** [Hanlard/Bert-for-WebQA#1]
请问你的这种数据预处理方式可以用于 BertForQuestionAnswering 的输入吗？

**Comment by Hanlard** [comment]
> 请问你的这种数据预处理方式可以用于 BertForQuestionAnswering 的输入吗？

是一样的，这是tansformers包里边BertForQuestionAnswering的demo：
    Examples::

        tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
        model = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad')
        question, text = "Who was Jim Henson?", "Jim Henson was a nice puppet"
        input_text = "[CLS] " + question + " [SEP] " + text + " [SEP]"
        input_ids = tokenizer.encode(input_text)
        token_type_ids = [0 if i <= input_ids.index(102) else 1 for i in range(len(input_ids))]
        start_scores, end_scores = model(torch.tensor([input_ids]), token_type_ids=torch.tensor([token_type_ids]))
        all_tokens = tokenizer.convert_ids_to_tokens(input_ids)
        print(' '.join(all_tokens[torch.argmax(start_scores) : torch.argmax(end_scores)+1]))
        # a nice puppet
